run/invoker.go

```diff
@@ -169,7 +169,8 @@ func invoke(runenv *runtime.RunEnv, fn interface{}) {
 		}
 	case p := <-panicHandler:
 		// propagate the panic.
-		panic(p)
```

Comment:
I am not entirely sure why we want to panic again here. Isn't just `RecordCrash` enough at this point?